### ISSUE TITLE
Basic customizer options

### ISSUE THREAD

**Issue by rubengc** [chrisblakley/Nebula#1194]
I am working on add support for WordPress customizer modifing basic thoughts (like logo, title, descripción, icon and primary/secondary/background colors)

On child themes, adding a filter to customizer anyone could override parent theme customizer

So If you like this I can PR when I finish this

**Comment by chrisblakley** [comment]
Feel free to make a pull request for Customizer support- I may piecemeal the functionality in some cases. For example, color options will need to also feed my Sass compiler to set default color values (which are currently hard-coded).

If you want to make a PR for it, I can handle some of the deep-seeded functionality.

**Comment by chrisblakley** [comment]
Cool, thanks for the PR- I'll check this out and commit it soon.

**Comment by chrisblakley** [comment]
Here's what I did after merging that PR:
- Opted to use Sass variables as default color values (in nebula_utilities.php in `nebula_render_scss`)
  - I like the idea of fewer embedded styles as possible and this gives us better control of overwriting
- I then moved the register function to nebula_functions.php.
  - If it gets big enough for it's own file we can move it again. I thought about moving it to nebula_admin.php, but for now it's fine.

Some ideas for this:
- Possibly trigger Sass render when Customizer is saved (right now it needs to be manually processed to show changes).
- Might add text color with a default of `#444` in the future.
